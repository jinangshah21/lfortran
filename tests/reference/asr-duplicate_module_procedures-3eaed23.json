{
    "basename": "asr-duplicate_module_procedures-3eaed23",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/duplicate_module_procedures.f90",
    "infile_hash": "3a9e11036265ddc79e565af9c00310f2d8d7235a45c7c805375832bd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-duplicate_module_procedures-3eaed23.stdout",
    "stdout_hash": "f7f3bac9a32ba40a4abadf2aac009909291f583a780054a6ff9af424",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}