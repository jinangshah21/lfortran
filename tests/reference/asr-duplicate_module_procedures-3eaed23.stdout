(TranslationUnit
    (SymbolTable
        1
        {
            input_module:
                (Module
                    (SymbolTable
                        2
                        {
                            test_01:
                                (Function
                                    (SymbolTable
                                        3
                                        {
                                            x:
                                                (Variable
                                                    3
                                                    x
                                                    []
                                                    In
                                                    ()
                                                    ()
                                                    Default
                                                    (Integer 4)
                                                    ()
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    test_01
                                    (FunctionType
                                        [(Integer 4)]
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    []
                                    [(Var 3 x)]
                                    [(Print
                                        (StringFormat
                                            ()
                                            [(Var 3 x)]
                                            FormatFortran
                                            (Character -1 0 () PointerString)
                                            ()
                                        )
                                    )]
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                ),
                            test_interface:
                                (GenericProcedure
                                    2
                                    test_interface
                                    [2 test_01
                                    2 test_01
                                    2 test_01]
                                    Public
                                )
                        })
                    input_module
                    []
                    .false.
                    .false.
                ),
            main:
                (Program
                    (SymbolTable
                        4
                        {
                            test_01:
                                (ExternalSymbol
                                    4
                                    test_01
                                    2 test_01
                                    input_module
                                    []
                                    test_01
                                    Public
                                ),
                            test_interface:
                                (ExternalSymbol
                                    4
                                    test_interface
                                    2 test_interface
                                    input_module
                                    []
                                    test_interface
                                    Public
                                ),
                            test_interface@test_01:
                                (ExternalSymbol
                                    4
                                    test_interface@test_01
                                    2 test_01
                                    input_module
                                    []
                                    test_01
                                    Private
                                )
                        })
                    main
                    [input_module]
                    [(SubroutineCall
                        4 test_interface@test_01
                        4 test_interface
                        [((IntegerConstant 1 (Integer 4) Decimal))]
                        ()
                    )]
                )
        })
    []
)
